services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -457,8 +469,9 @@ public Response updateRealm(final RealmRepresentation rep) {
             throw ErrorResponse.error(e.getMessage(), Status.INTERNAL_SERVER_ERROR);
         } catch (ModelException e) {
             throw ErrorResponse.error(e.getMessage(), Status.BAD_REQUEST);
+        } catch (org.keycloak.services.ErrorResponseException e) {
+            throw ErrorResponse.error("AccessCodeLifespanLogin or AccessCodeLifespanUserAction cannot be 0", Status.BAD_REQUEST);
```

Comment:
Can you please update this line just to:
```
throw e
```
so that the original exception is just re-thrown?

As for example when some other `ErrorResponse` was thrown (like the `Can't rename realm`), then it would be incorrectly re-thrown with the error `AccessCodeLifespanLogin or AccessCodeLifespanUserAction cannot be 0`, which would be very confusing.